### PR TITLE
dnaro blog post, mar17

### PR DESCRIPTION
**Pull request by oraNod**
with a few hidden song references

### CODE REVIEW

File: blog/2020-03-17-documentation-strategy.adoc [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+the right way or when to set them, then it's just giving users a long rope to
+go hang themselves with.
+
+So hopefully one of the biggest things you'll notice in the Infinispan
+documentation is that we're adding more procedures. We want to give you the
+recommended fast path that gets you up and running as quickly as possible.
+
+.Taking a topic-based approach
+
+Topics are logical units of information that can standalone and map to various
```
Comment (tristantarrant) [inline blog/2020-03-17-documentation-strategy.adoc]: shouldn't that be "stand alone" ?

File: blog/2020-03-17-documentation-strategy.adoc [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+At the same time topic-based writing reduces maintenance overhead. A huge win
+for topics is that they allow reuse. Write something once and use it in
+different contexts as needed.
+
+== Quick Tour of the Repository Layout
+
+Now that I've explained some of the motivation, I'd like to briefly delve into
+how we've organized the docs.
+
+Documentation source lives in `documentation/src/main/asciidoc` and has three
```
Comment (tristantarrant) [inline blog/2020-03-17-documentation-strategy.adoc]: Linkify this to the actual directory on github

File: blog/2020-03-17-documentation-strategy.adoc [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+main subdirectories:
+
+* `/topics`
```
Comment (tristantarrant) [inline blog/2020-03-17-documentation-strategy.adoc]: linkify

File: blog/2020-03-17-documentation-strategy.adoc [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
++
+Topics are all the different pieces of information that make up the
+documentation. Asciidoc files in this directory are prefixed with `proc_*` for
+procedural content, `con_*` for concepts, and `ref_*` for reference material.
++
+Apart from the prefixes, topic filenames should be readable and match headings
+as they appear in the rendered version.
+
+* `/stories`
```
Comment (tristantarrant) [inline blog/2020-03-17-documentation-strategy.adoc]: linkify

File: blog/2020-03-17-documentation-strategy.adoc [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
++
+These are asciidoc files prefixed with `assembly_*` that organize topics into
+complete, end-to-end sets of content that target specific use cases .
+
+* `/titles`
```
Comment (tristantarrant) [inline blog/2020-03-17-documentation-strategy.adoc]: linkify

File: blog/2020-03-17-documentation-strategy.adoc (hunk 1 above)
Comment (oraNod) [reply blog/2020-03-17-documentation-strategy.adoc]: it should. nice catch.

**Pull request merged**